server/integration_tests/test_data/demo2_cities_feb2023/query_2/chart_config.json

```diff
@@ -60,6 +60,34 @@
             ],
             "title": "Chronic Obstructive Pulmonary Disease"
           },
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "statVarKey": [
+                      "Percent_Person_WithArthritis"
```

Comment:
Wonder why this shows up now?

Reply:
Since the topic was removed, this is the additional stat var to fill the ranking list.

Reply:
Ah, ok